### ISSUE TITLE
View Blocked Domains

### ISSUE THREAD

**Issue by qauff** [s-s/dnscloak#7]
I’m trying to unblock an app, but I can’t seem to find which domain is being blocked. Is there a view or log that will show us the domains that are blocked? In the DNS queries log I only see domains marked as PASS. Maybe I’m looking in the wrong place.

**Comment by mikegchambers** [comment]
If the domain is being blocked by a Blacklist within the App, then you can enable and look at the Blacklist logs.  It will also show up as REJECTED in the DNS queries log.   If the domain is being blocked by a DNS provider (off app) then I'm not sure what you'll see.

**Comment by qauff** [comment]
@mikegchambers yeah, in other apps you can see all DNS requests, which will let you know which domains to whitelist. Or at least give you an idea, it’s just hard to see with this one cause I don’t know what was blocked
